src/js/MainController.js

```diff
@@ -114,7 +116,10 @@
             isWindowTooSmall()) {
           showCatalog = false;
         }
-        $scope.globals.catalogShown = showCatalog;
+        if (!initWithPrint) {
+          $scope.globals.catalogShown = showCatalog;
+          initWithPrint = false;
```

Comment:
On the next topi change the print widget stays open and the catalog is always closed. I don't know if it's wanted or not